### PR TITLE
feat(create-medusa-app): update command for a better onboarding experience

### PR DESCRIPTION
**Pull request by shahednasser**
## What

Updates the NPX `create-medusa-app` command for a better user and onboarding experience.

## Testing

- change to the `packages/create-medusa-app` directory
- install dependencies if necessary
- run `yarn dev`

### CODE REVIEW

File: packages/create-medusa-app/src/utils/prepare-project.ts [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    })
+
+    interval = resetFactBox(interval, spinner, "Created admin user")
+  }
+
+  if (seed) {
+    interval = createFactBox(spinner, "Seeding database...")
+
+    // check if a seed file exists in the project
+    if (!fs.existsSync(path.join(directory, "data", "seed.jsons"))) {
```
Comment (olivermrbl) [inline packages/create-medusa-app/src/utils/prepare-project.ts]: todo: 
```suggestion
    if (!fs.existsSync(path.join(directory, "data", "seed.json"))) {
```

File: packages/create-medusa-app/src/utils/facts.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  "Plugins allow you to integrate third-party services for payment, fulfillment, notifications, and more.",
+  "You can specify a product's availability in one or more sales channels.",
+  "Payment and shipping options and providers can be configured per region.",
+  "Tax-inclusive pricing allows you to set prices for products, shipping options, and more without having to worry about calculating taxes.",
+  "Medusa provides multi-currency and region support, with full control over prices for each currency and region.",
+  "You can organize customers by customer groups and set special prices for them.",
+  "You can specify the inventory of products per location and sales channel.",
+  "Publishable-API Keys allow you to send requests to the backend within a scoped resource.",
+  "You can create custom endpoints by creating a TypeScript file under the src/api directory.",
+  "You can listen to events to perform asynchornus actions using Subscribers.",
```
Comment (olivermrbl) [inline packages/create-medusa-app/src/utils/facts.ts]:
```suggestion
  "You can listen to events to perform asynchronous actions using Subscribers.",
```

**Review by olivermrbl — CHANGES_REQUESTED** [review]
this is a fantastic addition 🚀 🚀

File: packages/create-medusa-app/src/utils/prepare-project.ts (hunk 1 above)
Comment (shahednasser) [reply packages/create-medusa-app/src/utils/prepare-project.ts]: Nice catch 😅

**Review by olivermrbl — APPROVED** [review]
LGTM, great work!! 🚢

**Pull request merged**